### PR TITLE
[Urgent][Security Bugs]LPD-1558 add properties to control captcha on or off for ci

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

Resend: https://github.com/brianchandotcom/liferay-portal/pull/148936
Added a properties "system.captcha.enabled" to disable capture while runinng ci to avoid failing functional test.

### CODE REVIEW

**Comment by jeffwu0724** [comment]
ci:test:relevant

**Comment by jeffwu0724** [comment]
ci:test:acceptance-functional

**Comment by jeffwu0724** [comment]
ci:test:backend

**Comment by jeffwu0724** [comment]
ci:report:2560314016

**Comment by jeffwu0724** [comment]
No unique failure founds in releant test, waiting for others...

**Comment by jeffwu0724** [comment]
ci:help

**Comment by jeffwu0724** [comment]
ci:test:acceptance-functional

**Comment by jeffwu0724** [comment]
ci:test:backend

**Comment by jeffwu0724** [comment]
ci:test:backend

**Comment by jeffwu0724** [comment]
ci:report:2560474468

File: build-test.xml [hunk 1]
```diff
@@ -11893,6 +11893,8 @@ ${update.properties}</echo>
 
 liferay.home=${liferay.home}
 
+system.captcha.enabled=false
```
Comment (tinatian) [inline build-test.xml:11896]: @jeffwu0724 Please move this line the end of this file

**Comment by jeffwu0724** [comment]
ci:close

**Pull request closed without merging**